sample_factory/algo/runners/runner.py

```diff
@@ -505,9 +506,17 @@ def _setup_component_heartbeat(self, component: HeartbeatStoppableEventLoopObjec
             self.heartbeat_dict[component_type] = {}
         type_dict = self.heartbeat_dict[component_type]
         type_dict[component.object_id] = time.time()
+
+        """
```

Comment:
This is not how one usually uses docstrings. Docstrings are intended for functions and classes, if you want to add a comment in the middle of the function, just use regular comment starting with `#`